### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NewJobSteps_ScreenReader: NVDA announce "Blank" while changing value in "Retry Attempts" spin button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6028]
[Check out Keros! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
NVDA Version: 2019.1

**Additional Details:**
Tools Name: NVDA
MAS Number :MAS1.3.1

**Steps to Reproduce :**
1. Install the app Azure Data Studio.
2. Navigate  to the SQL agent tab from Home page->Click on new job button->Select  steps tab.
3. New job step screen opens.
4. Navigate to the "Advenced tab".
5. Start NVDA.
6. Navigate till "Retry Attempts" spin button.
7. Observe that when changing value for "Retry Attempts" spin button ,screen reader announces "blank".

**Actual:**
Navigate till "Retry Attempts" spin button by default value is 0 ,screen reader announce "0 spin button editable 0" . After changing this value to 1 using up and down arrow ,screen reader announces "blank 1 unselected" and continue to change value to 2 then it announces only "Blank" .After that if continues to change the value then only announce "Blank".

**Similar issue has been observed for below page -**
**1. CreateAlertOptions -** Go to the  SQL server agent tab->On alerts tab->Click on new alert button    ->Select Options tab.

**Expected:**
Screen reader should announce the only changed value and not "Blank". For example, If value is changed from 0 to 1 then screen reader should announce "1 unselected " . 

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader only user find it difficult to perform any action on the spin button because they don't understand which value is selected .

**MAS Reference:
MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[6028_[Accessibility]A11y_AzureDataStudio_NewJobSteps_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3290382/6028_.Accessibility.A11y_AzureDataStudio_NewJobSteps_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes
